src/bundle/Resources/config/routing.yml

```diff
@@ -8,3 +8,8 @@ ez_hybrid_platform_ui_version_draft_actions:
     defaults:
         _controller: EzSystemsHybridPlatformUiBundle:Version:draftActions
     methods: [POST]
+ez_hybrid_platform_ui_version_archive_actions:
+    path: /version/archive-actions/{contentId}
```

Comment:
I'd rather have `content` in the URI.
Maybe `/content/{contentId}/versions/archives` ?